hydra-node/src/Hydra/Options.hs

```diff
@@ -534,6 +534,38 @@ hydraNodeCommand =
       (decodeUtf8 $ encodePretty Contract.scriptInfo)
       (long "script-info" <> help "Dump script info as JSON")
 
+data InvalidOptions
+  = MaximumNumberOfPartiesExceeded
+  | CardanoAndHydraKeysMissmatch
+  deriving (Eq, Show)
+
+-- | Hardcoded limit for maximum number of parties in a head protocol
+-- NB: we don't count our own node here so need to do + 1 at the call site
```

Comment:
It does include our own node. In the calculation below we need to `+ 1` because we only look at the "other party" configurations. Let's remove this comment here and Fix it below.

Reply:
Can you also refer to where to find this value? The `tx-cost` executable prints the max possible parties for all the transitions. For example the `computeCollectComCost` should determine this maximum.